### PR TITLE
Site: Better Performance

### PR DESCRIPTION
**Pull request by DirtyF**
This is a 🔦 documentation change.

TL;DR: PageSpeed report went from 94 to 100

## Context

Current PageSpeed report: https://developers.google.com/speed/pagespeed/insights/?url=jekyllrb.com

pagespeed-current

Speed Index: 2.3s
Time to Interactive: 3.2s
First Meaningul Paint; 2.3s 

## Changes

Make Jekyll website load a little bit faster by:

- self-hosting fonts with modern formats (woff, woff2)
- preloading important fonts earlier 
- prioritize readable text
- self-hosting docsearch CSS to avoid network latency
- preloading main CSS to avoid blocking rendering
- resize and optimize logos to have a lighter total

New PageSpeed report: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fdeploy-preview-7388--jekyllrb.netlify.com%2F

pagespeed-7138

Speed Index: 1.3s
Time to Interactive: 2.4s
First Meaningul Paint; 1s

### CODE REVIEW

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Comment by parkr** [comment]
Nice work!

**Comment by DirtyF** [comment]
@Parker Thanks, next time we'll try to aim at a perfect score at Progressive Web App.

**Comment by ashmaroli** [comment]
Wow..!! :heart:
@DirtyF There's a typo in your comment above.. You pinged the wrong Parker.. :stuck_out_tongue_closed_eyes:
